### ISSUE TITLE
Error description window disappear instantly if start of the line is not on screen

### ISSUE THREAD

**Issue by MauricioFerraz** [palantir/tslint-playground#10]
### Bug Report

- __TSLint version__:
- __TypeScript version__:
- __Running TSLint via__: (pick one) CLI / Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc

#### Reproduction using [TSLint Playground](https://palantir.github.io/tslint-playground)
[tslint-playground](https://palantir.github.io/tslint-playground/?saved=N4Igxg9gJgpiBcID0SAEBlALgQwE6dUwE8AHASwDsBzVSwgCxlUllQCMIAPQiVKmAgGcc+GFAB0AHQrTsFIgBkI1BZRgB1MpnoBBCgFFcuCLlQBqVHMXKqqihq26DRk+cvylKtZu17Dx0wsrT1tvRz8XQPdrL3sfJ39XII8bOwdfZwCACkkQRNxBXIBKAG5pEAAacGUAMzIqBGQ0AE0IAFdmOVQAW2gyGqIGJkgKOqo23GxMMmVURlwmAHdHVCJ20whFiikKFFQAWWwAayZBCaZtKdX11AArQVmyQVQAN2wAGzIoCvcoWhrrh1cG0KLQKJgINI9hBtDBTE8zjBBD9MLhBmBGGAjpQaLDmMoHu8mDVXHCAoIdtJgNJULTULkYJxMDAKFBCghUABtXKYQSfcHwBaQbrdFmwKC5AC6FRpdNywKJ7PgqGAAF9UHtsFA-lqoFoZhQPqgFUjfkMyKYRmMJlMDdJVeVVUA)

#### TypeScript code being linted

just any long line with an error

with `tslint.json` configuration:

default

#### Actual behavior

  If you scroll a wide page to the right, to the point where you can't see the start of the line anymore, the error description window, that usually shows when you hover your mouse over the line, simply pops in and out of existence.

[clip showing the bug in action](https://drive.google.com/uc?id=1AH1C75KKnrmISN8DtXmmQ4uQ7F-O13Da)

#### Expected behavior

  The window should show regardless of the start of the line being in sight or not.

**Comment by MauricioFerraz** [comment]
I've noticed that it happens when the beginning of the highlighted portion of the code isn't on the screen:
[example running in Visual Studio Code](https://drive.google.com/uc?id=1RoeTqgLAbNADWewt6572aL_IQFIAOOW_)

**Comment by MauricioFerraz** [comment]
May I request that this issue be transferred back to palantir/tslint, since it doesn't happen only in the TSLint Playground?

**Comment by adidahiya** [comment]
tslint has no control over how its errors are displayed... this bug is specifically related to a GUI so I don't think it belongs in the tslint repo

**Comment by MauricioFerraz** [comment]
Yeah, I actually noticed that today. Even without tslint, vscode still shows the same behavior, so I'll try placing this issue there.
Thanks!

**Comment by MauricioFerraz** [comment]
On the other hand, the playground might still benefit from trying to fix it as well, so maybe closing it here was a bit rushed. 
Your call if it should be reopened.

And I've found out that they've fixed it on the insiders version.

**Comment by adidahiya** [comment]
We can probably get the same bugfix if we just update monaco-editor. Reopening to track that
